types/node-barion/index.d.ts

```diff
@@ -0,0 +1,267 @@
+// Type definitions for node-barion 3.1
+// Project: https://github.com/aron123/node-barion
+// Definitions by: Áron Kiss <https://github.com/aron123>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
```

Comment:
```suggestion
```
this is only required here, due to `console` methods being used in the tests. 
Those can be removed completely or replaced with proper type assertions:
https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs

Reply:
word of explanation: each such dependency put  another dependency test on @types/node CI tests, which are run on any @types/node definition changes. More explicit dependencies, longer tests, etc. Your types do not use any of Node native symbols, requiring explicit dependency on @types/node, just language features.